dimod/binary_quadratic_model.py

```diff
@@ -106,37 +133,43 @@ class assume that they are numeric.
             adjacent to `v` (e.g. `u`) and values are quadratic biases associated
             with the pair of inner and outer keys (`u, v`).
 
-            Examples:
-               This example creates an instance of the BinaryQuadraticModel()
-               class for the K4 complete graph, where the nodes have biases
-               set equal to their sequential labels and interactions are the
-               concatenations of the node pairs (e.g., 23 for u,v = 2,3).
-
-               >>> import dimod
-               >>> linear = {1: 1, 2: 2, 3: 3, 4: 4}
-               >>> quadratic = {(1, 2): 12, (1, 3): 13, (1, 4): 14,
-               ...              (2, 3): 23, (2, 4): 24,
-               ...              (3, 4): 34}
-               >>> offset = 0.0
-               >>> vt = dimod.BINARY
-               >>> bqm_k4 = dimod.BinaryQuadraticModel(linear, quadratic, offset, vt)
-               >>> bqm_k4.adj.viewitems()   # Show all adjacencies  # doctest: +SKIP
-               dict_items([(1, {2: 12, 3: 13, 4: 14}),
-                           (2, {1: 12, 3: 23, 4: 24}),
-                           (3, {1: 13, 2: 23, 4: 34}),
-                           (4, {1: 14, 2: 24, 3: 34})])
-               >>> bqm_k4.adj[2]            # Show adjacencies for node 2
-               {1: 12, 3: 23, 4: 24}
-               >>> bqm_k4.adj[2][3]         # Show the quadratic bias for nodes 2,3
-               23
-
         info (dict):
-            A place to store miscellaneous data about the BinaryQuadraticModel as a whole.
+            A place to store miscellaneous data about the :class:`.BinaryQuadraticModel`
+            as a whole.
 
         SPIN (:class:`.Vartype`): An alias of :class:`.Vartype.SPIN` for easier access.
 
         BINARY (:class:`.Vartype`): An alias of :class:`.Vartype.BINARY` for easier access.
 
+    Examples:
+       This example creates an instance of the :class:`.BinaryQuadraticModel`
+       class for the K4 complete graph, where the nodes have biases
+       set equal to their sequential labels and interactions are the
+       concatenations of the node pairs (e.g., 23 for u,v = 2,3).
+
+       >>> import dimod
+       >>> linear = {1: 1, 2: 2, 3: 3, 4: 4}
+       >>> quadratic = {(1, 2): 12, (1, 3): 13, (1, 4): 14,
+       ...              (2, 3): 23, (2, 4): 24,
+       ...              (3, 4): 34}
+       >>> offset = 0.0
+       >>> vt = dimod.BINARY
```

Comment:
Should spell our `vartype`. We aren't paying by the character :smile: . It keeps it consistent with the input argument names and ideally I should be able to know just from the name what each variable does.

Reply:
Fixed